web/src/message_scroll.js

```diff
@@ -14,10 +14,23 @@ import * as unread_ui from "./unread_ui";
 let hide_scroll_to_bottom_timer;
 export function hide_scroll_to_bottom() {
     const $show_scroll_to_bottom_button = $("#scroll-to-bottom-button-container");
+    $("#scroll-to-bottom-button-clickable-area").off("mouseenter");
```

Comment:
Why we need this?

Reply:
This is done to ensure that no previous "mouseenter" event handlers are active.
When I try to remove this , everytime there is a mouseenter on the given element , the function runs multiple times.